src/main/java/org/mozilla/magnet/scanner/MagnetScannerItem.java

```diff
@@ -4,56 +4,105 @@
 import org.json.JSONObject;
 
 import java.util.HashMap;
+import java.util.Map;
 
 public class MagnetScannerItem {
     private double mDistance = -1;
+    private String mChannelId;
+    private String mImageUri;
```

Comment:
I understand that we are adding this to try to save a call to the metadata service, but will it be enough? What if we want to add description too?
Should we just add the channel id and relay on a second call?